Usage.md

```diff
@@ -56,15 +56,15 @@ jobs:
 
   test-self-hosted:
     name: Testing self-hosted tag
-    needs: start-runner
+    needs: start-runners
```

Comment:
Can't comment directly, but might as well bump the version of the `configure-aws-credentials` action to `v4` in the example while whe're here

Reply:
Done.